### ISSUE TITLE
Issue with image larger than container div

### ISSUE THREAD

**Issue by samcackett** [timmywil/panzoom#92]
Hi there,

First just want to say that this plugin is awesome! Thank you for all your hard work. 

Unfortunately I just cannot get it working when using "container:invert"  - not sure if this is a bug or I am implementing it wrong. 

I have a panzoom container div with an image inside (image is loaded through javascript):

<div class='assettopview' id='substationview'>
                
                <section id="mapdiv" class="parent">
                    <div class="panzoom">
                      <img id="mapcad" src="" />
    
                    </div>

```
            </section>
            <section id="zoomInDiv" class="zoombuttons">
              <button type="button" id="mapcad_zoomin" data-role="none" class="zoom-in zoombutton"><img class="zoombtnimg" src="images/map-icon-zoomin.png" /></button>
            </section>
            <section id="zoomOutDiv" class="zoombuttons">  
              <button type="button" id="mapcad_zoomout" data-role="none" class="zoom-out zoombutton"><img class="zoombtnimg" src="images/map-icon-zoomout.png" /></button>
            </section>
            <section id="mapButtonDiv" class="zoombuttons">  
              <button type="button" class="zoombutton" id="map_imgbtn" data-role="none"><img id="map_img" class="zoombtnimg" src="images/map-icon-map.png" /></button>
            </section>
            <section id="cadButtonDiv" class="zoombuttons">
              <button type="button" class="zoombutton" id="cad_imgbtn" data-role="none"><img id="cad_img" class="zoombtnimg" src="images/map-icon-cad.png" /></button>
            </section>

        </div>
```

And then the panzoom function as so:

$('#substationview').find('.panzoom').panzoom({
                  increment: 0.2,
                  minScale: 1,
                  maxScale: 7,
                  duration: 200,
                  transition: true,
//                startTransform: "matrix(1.4, 0, 0, 1.4, 0, 0)",
                  contain: 'invert', // SC - this prevents image from scrolling outside of background, not fully working however
                  $zoomIn: $('#substationview').find(".zoom-in"),
                  $zoomOut: $('#substationview').find(".zoom-out"),
                  $zoomRange: $('#substationview').find(".zoom-range"),
//                $reset: $('#substationview').find(".reset")
                });

And what I am finding is it works as expected without contain: invert. 

With contain: invert, if the image is larger than the container div, panzoom correctly figures out the height and allows me to move the image up and down, but for the width, it does not let me move the image at all. 

In other words Y-axis is fine, X-axis doesn't budge from zero in the transform matrix (unless the image is zoomed in). 

Tested on Firefox, Chrome, Safari. 

Have I just  implemented this wrong or is this an issue? 

Many thanks,

Sam Cackett
